sympy/physics/continuum_mechanics/truss.py

```diff
@@ -0,0 +1,384 @@
+        [('AB', 'A', 'B'), ('BC', 'B', 'C')]
+        """
+        if label not in self._member_labels:
+            raise ValueError("No such member exists in the Truss")
+
+        else:
+            self._members.remove((label, self._member_nodes[label][0], self._member_nodes[label][1]))
+            self._member_labels.remove(label)
+            self._member_nodes.pop(label)
+
+    def apply_load(self, location, magnitude, direction):
+        """
+        This method applies an external load at a particular node
+
+        Parameters
+        ==========
+        location: String or Symbol
+            Label of the Node at which load is applied.
+
+        magnitude: Sympifyable
+            Magnitude of the load applied. It must always be positive and any changes in
+            the direction of the load are not reflected here.
+
+        direction: Sympifyable
+            The angle, in degrees, that the load vector makes with the horizontal. It takes
+            the values 0 to 360, inclusive.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> from sympy import symbols
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> P = symbols('P')
+        >>> t.apply_load('A', P, 90)
+        >>> t.apply_load('A', P/2, 45)
+        >>> t.apply_load('A', P/4, 90)
+        >>> t.loads
+        {'A': [[5*P/4, 90], [P/2, 45]], 'B': [[0, 90]]}
+        """
+        magnitude = sympify(magnitude)
+        direction = sympify(direction)
+
+        if location not in self.node_labels:
+            raise ValueError("Load must be applied at a known node")
+
+        else:
+            if direction in [self._loads[location][i][1] for i in range(len(self._loads[location]))]:
+                for i in range(len(self._loads[location])):
+                    if self._loads[location][i][1] == direction:
+                        self._loads[location][i][0] += magnitude
+                        break
+            else:
+                self._loads[location].append([magnitude, direction])
+
+    def add_support(self, location, type):
+        """
+        This method adds a pinned or roller support at a particular node
+
+        Parameters
+        ==========
+
+        location: String or Symbol
+            Label of the Node at which support is added.
+
+        type: String
+            Type of the support being provided at the node.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> t.add_support('A', 'pinned')
+        >>> t.supports
+        {'A': 'pinned', 'B': 'none'}
+        """
+        if location not in self._node_labels:
+            raise ValueError("Support must be added on a known node")
+
+        else:
+            self._supports[location] = type
```

Comment:
Add reaction forces according to the support. The names can be 'R'+(node name)+(x/y)